v22.1/export.md

```diff
@@ -73,6 +80,35 @@ Successful `EXPORT` returns a table of (perhaps multiple) files to which the dat
 `rows` | The number of rows exported to this file.
 `bytes` | The file size in bytes.
 
+## Parquet types
+
+CockroachDB types map to [Parquet types](https://github.com/apache/parquet-format/blob/master/LogicalTypes.md) as per the following:
+
+| CockroachDB Type    | Parquet Type | Parquet Logical Type |
+--------------------|--------------|----------------------
+| [`BOOL`](bool.html) | `BOOLEAN` | `nil` |
+| [`STRING`](string.html) | byte array | `STRING` |
+| [`COLLATE`](collate.html) | byte array | `STRING` |
+| [`INET`](inet.html) | byte array | `STRING` |
+| [`JSONB`](jsonb.html) | byte array | `JSON` |
+| [`INT`](int.html) | `INT64` | `nil` |
```

Comment:
forgot this in the commit message: CRDB `INT4` and `INT2` are encoded as parquet type `INT32` (all other int aliases (INT and INT8) are encoded as parquet type `INT64`). CRDB FLOAT4 gets encoded as Parquet FLOAT32 and other aliases (FLOAT and FLOAT8) get encoded as parquet type FLOAT64